pkg/logs/internal/tailers/file/rotate_windows.go

```diff
@@ -39,6 +39,7 @@ func (t *Tailer) DidRotate() (bool, error) {
 	offset := t.lastReadOffset.Load()
 
 	if sz < offset {
+		log.Debugf("File rotation detected due to size change, fileSize=%d < lastReadOffset=%d", sz, offset)
```

Comment:
Just for consistency with the same message on Unix:
```suggestion
		log.Debugf("File rotation detected due to size change, lastReadOffset=%d, fileSize=%d", offset, sz)
```